test/integration/react-profiling-mode/next.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  reactProductionProfiling: true,
+  reactProductionProfiling: process.env.REACT_PRODUCTION_PROFILING === 'true',
```

Comment:
Let's use a `TEST_` prefix indicating this is for testing only
```suggestion
  reactProductionProfiling: process.env.TEST_REACT_PRODUCTION_PROFILING === 'true',
```